features/devicepixelratio.yml

```
@@ -0,0 +1,6 @@
+name: devicepixelratio
+description: The `Window.devicePixelRatio` read-only property returns the ratio of the vertical size of one physical pixel on the current display device to the size of one CSS pixel.
```

```suggestion
description: The `window.devicePixelRatio` read-only property returns the ratio of the size of one CSS pixel to the vertical size of one physical pixel on the current display device.
```

I _think_ the ratio goes the other way, but double check me.

---

Yeah it's funky to describe. I basically copied the Caniuse description because MDN's made things more muddy.

Yes, it's "physical" pixel / CSS pixel so i'll update to your suggestion which makes that clearer.

For reference, here's the spec:

> The devicePixelRatio attribute must return the result of the following determine the device pixel ratio algorithm:
> 
> - If there is no output device, return 1 and abort these steps.
> - Let CSS pixel size be the size of a [CSS pixel](https://drafts.csswg.org/css-values-4/#px) at the current [page zoom](https://drafts.csswg.org/cssom-view/#page-zoom) and using a [scale factor](https://drafts.csswg.org/cssom-view/#scale-factor) of 1.0.
> - Let device pixel size be the vertical size of a device pixel of the output device.
> - Return the result of dividing CSS pixel size by device pixel size.

Boris and PPK's write-ups are good too:

- https://web.dev/articles/high-dpi
- https://www.quirksmode.org/blog/archives/2012/06/devicepixelrati.html